crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -478,7 +482,7 @@ pub fn check_visibility<QF>(
                 }
 
                 // If we have an aabb, do frustum culling
-                if !no_frustum_culling {
+                if !no_frustum_culling && !no_cpu_culling {
```

Comment:
Maybe it's just me but those double negative are a bit confusing. Would it make sense to use `!gpu_culling` instead? I guess there would still the the `!no_frustum_culling` so that might be even more confusing to use something different after.

Reply:
Well, "no CPU culling" doesn't imply "GPU culling". You can have GPU culling and CPU culling active at the same time. This isn't useful now, but once we have GPU occlusion culling, it can be useful. It might even be the default, depending on whether GPU occlusion culling or retained bins lands first. That's because CPU culling is very useful to reduce the amount of CPU load in `queue_material_meshes`, which will continue to be expensive at least until retained bins lands.

Reply:
> "no CPU culling" doesn't imply "GPU culling".

Right, yeah, I didn't consider that